README.md

```diff
@@ -1 +1,142 @@
-# node-io
\ No newline at end of file
+# Basic Node I/O
+
+## Overview
+
+This lesson will cover the ...
+
+## Objectives
+
+1. Illustrate the basic async code (non-blocking I/O) vs. sync code, e.g., `setTimeout()` in Node
+2. Describe `process.nextTick()`
+3. Describe `setImmediate()`
+
```

Comment:
Need an intro - I think the  Ruby example would be good -  of  synchronous code is great to explain why node I/O is great with callbacks to run asynchronously